packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -25,29 +25,29 @@ import {
 import fetch from "cross-fetch";
 import safeStringify from "json-stringify-safe";
 
-import type { AxiosRequestConfig, RawAxiosRequestHeaders } from "./axios.cjs";
 import {
 	getUrlForTelemetry,
 	RouterliciousErrorTypes,
 	throwR11sNetworkError,
 } from "./errorUtils.js";
 import { pkgVersion as driverVersion } from "./packageVersion.js";
 import { addOrUpdateQueryParams, type QueryStringType } from "./queryStringUtils.js";
+import type { RequestConfig, RawRequestHeaders } from "./request.cjs";
 import { RestWrapper } from "./restWrapperBase.js";
 import { ITokenProvider, ITokenResponse } from "./tokens.js";
 
 type AuthorizationHeaderGetter = (token: ITokenResponse) => string;
 export type TokenFetcher = (refresh?: boolean) => Promise<ITokenResponse>;
 
-const buildRequestUrl = (requestConfig: AxiosRequestConfig): string =>
+const buildRequestUrl = (requestConfig: RequestConfig) =>
```

Comment:
```suggestion
const buildRequestUrl = (requestConfig: RequestConfig): string =>
```
We'll still need the return type at some point anyway.